crates/sui/tests/checkpoints_tests.rs

```diff
@@ -235,6 +235,10 @@ async fn checkpoint_with_shared_objects() {
                 long_pause_between_checkpoints: Duration::from_millis(10),
                 ..CheckpointProcessControl::default()
             };
+
+            println!("Start active execution process.");
+            active_state.clone().spawn_execute_process().await;
```

Comment:
How is it possible that the test was working before you added these line of code? It seems that the test should have failed 100% of the time without it.

Reply:
Hm, because sometimes the node happens to have processed the transaction?

Reply:
mmh, I remember running this test many times to make sure it wasn't flaky.. will run for more next time